project/ws/viewer/src/lib/components/viewer-toc/viewer-toc.component.html

```diff
@@ -54,9 +54,16 @@ <h2 class="text-truncate margin-remove">Course Content</h2>
             </ws-widget-display-content-type-icon> -->
               <span role="button" [ngClass]="{ 'content-active': pathSet.has(node.identifier) }" matRipple
                 matTreeNodeToggle class="margin-left-xs mt-2 width-9-10">
+
                 <h3 class="text-truncate margin-remove">{{ node.title }}</h3>
                 <div class="mb-3 width-max-content">
-                  Total: {{ node.duration | pipeDurationTransform: 'hms' }}
+                  Total: {{ node.duration | pipeDurationTransform: 'hms' }} |
+                  <span class="content-top" *ngIf="node?.progress * 100 === 100; else elseIfBlock">
```

Comment:
use relative naming for else block, instead of generic

Reply:
Fixed relative naming for else part.